client/src/components/Card/PokemonCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { Card } from "../../types/card/card.types";
+import { StatsList } from "../StatsList/StatsList";
+
+type PokemonCardProps = {
+  pokemon: Card;
+  isFlipped: boolean;
+  playerId: string;
+  playRound: () => void;
+};
+
+export const PokemonCard: React.FC<PokemonCardProps> = ({
+  pokemon,
+  isFlipped,
```

Comment:
Could we rename isFlipped here please? Maybe just to isShown? or isStatsShown? I don't know if we'll have time to do a flipping card animation specifically, so this way it's just letting us know that the key difference is showing the stats on the card or not.